shell/components/formatter/InternalExternalIP.vue

```diff
@@ -2,19 +2,60 @@
 import { isV4Format, isV6Format } from 'ip';
 import CopyToClipboard from '@shell/components/CopyToClipboard';
 import { mapGetters } from 'vuex';
+import RcStatusBadge from '@components/Pill/RcStatusBadge/RcStatusBadge';
+
 export default {
-  components: { CopyToClipboard },
+  components: { CopyToClipboard, RcStatusBadge },
   props:      {
     row: {
       type:     Object,
       required: true
     },
   },
   computed: {
+    ...mapGetters({ t: 'i18n/t' }),
+    filteredExternalIps() {
+      return this.row.externalIps?.filter((ip) => this.isIp(ip)) || [];
+    },
+    filteredInternalIps() {
+      return this.row.internalIps?.filter((ip) => this.isIp(ip)) || [];
+    },
     internalSameAsExternal() {
-      return this.row.internalIp === this.row.externalIp;
+      return this.filteredInternalIps[0] === this.filteredExternalIps[0];
```

Comment:
I think this computed property needs to be tweaked a bit. We're now showing 	
`- / Same as External` when neither category of IP is defined; I think we still want to show `- / -`
before
<img width="2134" height="358" alt="Screenshot 2025-10-24 at 4 52 14 PM" src="https://github.com/user-attachments/assets/8f2df136-9d0e-4ab0-8f4f-471cd2b441d3" />

after
<img width="1399" height="463" alt="Screenshot 2025-10-24 at 4 52 21 PM" src="https://github.com/user-attachments/assets/06d2ec8b-cbf7-43af-9794-f179bbd422b3" />